packages/generator/src/type/__tests__/__snapshots__/TypeGenerator.spec.js.snap

```diff
@@ -77,9 +82,9 @@ export default new GraphQLObjectType({
       resolve: obj => obj.title,
     },
     user: {
-      type: GraphQLID,
+      type: UserType,
       description: \'User that created this post\',
-      resolve: obj => obj.user,
+      resolve: async (obj, args, { user }) => await UserLoader.load(user, obj.user),
```

Comment:
On user we pass the entire context instead of just the `user`, should we work on it somehow?